build.gradle

```diff
@@ -1,6 +1,6 @@
 plugins {
     id "java"
-    id "edu.wpi.first.GradleRIO" version "2025.1.1"
+    id "edu.wpi.first.GradleRIO" version "2025.2.1"
```

Comment:
Most of the PRs today have this upgrade as well. Just something of note.

Reply:
> Most of the PRs today have this upgrade as well. Just something of note.

All the better to do it once here, and have everybody merge it sooner